src/Main.elm

```diff
@@ -28,7 +28,7 @@ green =
 
 main : Program () Model Msg
 main =
-    Browser.sandbox { init = init, update = update, view = view, apps = apps }
+    Browser.element { init = init, update = update, view = view, subscriptions = \_ -> Sub.none }
```

Comment:
To talk to the outside world (i.e. HTTP) you have to switch from using `Browser.sandbox`:

https://github.com/elm/browser/blob/d1dd66740615ac872c7cb7ddd9c3aa83d6d288a6/src/Browser.elm#L45

> Browser.sandbox: Create a “sandboxed” program that cannot communicate with the outside
world.

... to at least `Browser.element`:

> Browser.element Unlike a sandbox, an `element` can talk to the outside world...

Reply:
I'm guessing that `subscriptions = \_ -> Sub.none` is an inline declaration of apps (or subscriptions that you changed it to) and I looked up the `\_` and I think that is referring to an anonymous function definition?  I didn't realize that subscriptions were a known [thing](https://elmprogramming.com/subscriptions.html) and not just a variable name, so I'll definitely change back to that. I'm going to try doing:

```elm
subscriptions _ =
    Sub.none
```
and keep the subscriptions = subscriptions and see if that works.

Reply:
Yeah, `\_ -> Sub.none` is a short-hand to say there are no subscriptions for this element. 

The `\x -> <whatever>` is Elm's syntax for an anonymous function (aka lambdas in Python): https://elm-lang.org/docs/syntax#functions

The underscore as parameter to a function means you don't care about the actual value of that parameter.

I usually prefer named functions like you did with `subscriptions`. Nice job! Good instinct for readability and extensibility there. 💪 

And yes, subscriptions is a thing. The `Browser.element` function takes one parameter - a record with specific fields, one of which _has_ to be `subscriptions`:

https://github.com/elm/browser/blob/d1dd66740615ac872c7cb7ddd9c3aa83d6d288a6/src/Browser.elm#L104

When you changed the `subscriptions` field to `apps` you changed the type of the record, and the compiler would have complained.

More info about Elm records:

- https://elm-lang.org/docs/syntax#records
- https://elm-lang.org/docs/records